dashboard/config/linux/main.yml

```diff
@@ -80,6 +80,7 @@ includes:
  - kcsan.yml: [kcsan]
  - kmsan.yml: [kmsan]
  - kfence.yml: [kfence]
+ - baseline.yml: [baseline, -cuttlefish, -cos]
```

Comment:
Let's add a new tag in the tool that will enable only this fragment, then won't need -cuttlefish and -cos.